File: JavaScript/112A.js
"use strict";

const fs = require('fs');

/**
 * Problem: Codeforces 112A - Petya and Strings
 * 
 * Logic: Compare two strings lexicographically, ignoring case.
 * Output -1 if first is smaller, 1 if second is smaller, 0 if equal.
 */

function solve() {
    const input = fs.readFileSync(0, 'utf8').split(/\s+/);
    if (input.length < 2) return;
    
    // Case-insensitive comparison using toLowerCase()
    const string1 = input[0].toLowerCase();
    const string2 = input[1].toLowerCase();
    
    if (string1 < string2) {
        console.log("-1");
    } else if (string1 > string2) {
        console.log("1");
    } else {
        console.log("0");
    }
}

solve();

File: Python/112A.py
import sys

def solve():
    # Read the two input strings
    try:
        s1 = sys.stdin.readline().strip()
        s2 = sys.stdin.readline().strip()
    except EOFError:
        return

    if not s1 or not s2:
        return

    # Convert both to lowercase for case-insensitive comparison
    s1 = s1.lower()
    s2 = s2.lower()

    # Compare lexicographically
    if s1 < s2:
        print("-1")
    elif s1 > s2:
        print("1")
    else:
        print("0")

if __name__ == "__main__":
    solve()